sentences/pt/cover_HassTurnOff.yaml

```diff
@@ -8,12 +8,27 @@ intents:
         slots:
           device_class: garage
           domain: cover
+
       - sentences:
-          - <fecha>  [as|os|a|o] [(cortina|cortinas|cortinado|cortinados|estore|estores|persiana|persianas)] <na_zona>
+          - <fecha>  [as|os|a|o] <estores> <na_zona>
         response: cover
         slots:
           device_class:
             - blind
             - curtain
             - shutter
           domain: cover
+
+      # fechar estores na mesma divisão que um dispositivo de voz
+      - sentences:
+          - <fecha>  [as|os|a|o] <estores> [aqui]
+        response: cover
+        slots:
+          device_class:
```

Comment:
I think we can remove the device_class slot. 
Can you also remove them from the other rules please?

Reply:
You are correct. Changed. Thanks.